introduction.md

```diff
@@ -3,5 +3,9 @@ Welcome to my fork of the dojo_rule repo.
 This has been set up just to git the dojo_rules a whirl.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?